projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/case-file-view-field.component.spec.ts

```diff
@@ -1,17 +1,40 @@
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { ActivatedRoute } from '@angular/router';
+import { RouterTestingModule } from '@angular/router/testing';
+import { of } from 'rxjs';
+import { CaseFileViewService } from '../../../services';
 import { CaseFileViewFieldComponent } from './case-file-view-field.component';
+import createSpyObj = jasmine.createSpyObj;
 
 describe('CaseFileViewFieldComponent', () => {
   let component: CaseFileViewFieldComponent;
   let fixture: ComponentFixture<CaseFileViewFieldComponent>;
+  let mockCaseFileViewService: any;
```

Comment:
type should be jasmine.SpyObj<CaseFileViewService>

Reply:
thank you @MateiHMCTS I will do this as part of the next in progress ticket https://tools.hmcts.net/jira/browse/EUI-5377

Reply:
@johnbenjamin-hmcts - I've already corrected this in my PR, so it'll be there when it gets merged